Makefile

```diff
@@ -75,6 +75,17 @@ else # GTEST case
   SRCS := $(shell find $(SRC_DIRS) -type f \( -name '*.cpp' -o -name '*.c' \) | grep -v "$(EXCLUDE_DIRS)")
 endif
 
+# Check if BUILD_WEAK_STUBS_SRC is defined
```

Comment:
this is a separate ticket, and unrelated to autogenerate and should be on a separate branch.

Because you need to upgrade the testing suites for this to test being enabled and disabled.

Reply:
the autogenerate code is ready to merge, but this code is holding the ticket open.

Reply:
well, I would slightly disagree. We can totally put the Makefile changes to a different PR, but the autogenerate script  makes weak stubs in this PR.So, it's kind of related in that context.

Reply:
Created : https://github.com/rdkcentral/ut-core/pull/163